boards/e104-bt5010a-tb/include/periph_conf.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (C) 2020 Benjamin Valentin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_e104-bt5010a-tb
+ * @{
+ *
+ * @file
+ * @brief       Peripheral configuration for the E104-BT5010A Test Board
+ *
+ * @author      Benjamin Valentin <benpicco@googlemail.com>
+ *
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "periph_cpu.h"
+#include "cfg_rtt_default.h"
+#include "cfg_timer_default.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Clock configuration
+ *
+ * @note    The radio will not work with the internal RC oscillator!
+ *
+ * @{
+ */
+#define CLOCK_HFCLK         (0U)            /* set to  0: internal RC oscillator
+                                             *        32: 32MHz crystal */
+#define CLOCK_LFCLK         (0)             /* set to  0: internal RC oscillator
```

Comment:
Can you add `ifndef` guards here?

Reply:
Those are physical properties of the board (module).
 There is neither a 32kHz oscillator nor a 32 MHz one present on the module, so no point in overwriting this.

Reply:
Ah ok, can you fix the comment then? The comment on the side made me think they where configurable.

Reply:
Uh yea

> The radio will not work with the internal RC oscillator!

is obviously wrong.